lib/rex/post/meterpreter/extensions/stdapi/corrm/apps.rb

```diff
@@ -0,0 +1,77 @@
+# -*- coding: binary -*-
```

Comment:
Why don't we name this file `lib/rex/post/meterpreter/extensions/appapi/appapi.rb`?